### ISSUE TITLE
Prepass fails when running into a useLayoutEffect hooks

### ISSUE THREAD

**Issue by BartWaardenburg** [sventschui/preact-ssr-prepass#4]
I've created a fork with a failing test: https://github.com/BartWaardenburg/preact-ssr-prepass

I’m running into issues when using preact-ssr-prepass since it fails in combination with preact/hooks (most likely the useLayoutEffect hook). 

I’m getting the following error:
```
Server Error
TypeError: Cannot read property 'push' of undefined
This error happened while generating the page. Any console logs will be displayed in the terminal window.
Call Stack
x
file:///Users/bartwaardenburg/Sites/ANWB/poncho-next/node_modules/preact/hooks/dist/hooks.js (1:543)
y.Image
webpack-internal:///./node_modules/@anwb/image/src/Image.js (67:71)
l
file:///Users/bartwaardenburg/Sites/ANWB/poncho-next/node_modules/preact-ssr-prepass/dist/index.js (1:968)

```
I am able to 'fix' this by doing the following change inside the preact/hooks dist:
```js
- t.__h.push(i);
+ t.__H.__h.push(i);
```
Which sources inside the useLayoutEffect hook (_renderCallbacks is undefined) and I think the __H is the mangled .hooks property:
```js
export function useLayoutEffect(callback, args) {
  /** @type {import('./internal').EffectHookState} */
  const state = getHookState(currentIndex++, 4);
  if (!options._skipEffects && argsChanged(state._args, args)) {
    state._value = callback;
    state._args = args;
    currentComponent._renderCallbacks.push(state);
  }
}
```
Is there any way we can skip the useLayoutEffects?

**Comment by BartWaardenburg** [comment]
@JoviDeCroock pointed me to look at setting skipEffects inside options.

```js
options.__s = true;
options.skipEffects = true;

```
https://github.com/preactjs/preact/blob/master/hooks/src/index.js#L167

But that didn't seem to make any difference.

**Comment by sventschui** [comment]
Just published `1.0.2` that fixes this issue
